lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -367,7 +367,7 @@ module.exports = class MethodMappings {
         },
 
         async locateMultipleElements(element) {
-          const locator = Locator.create(element);
+          const locator = Locator.create(element, this.transport.api.isAppiumClient());
```

Comment:
I suggest to create a subclass of Locator which overrides the `create()` method. That way we don't have to pass the isAppium flag around and pollute the Locator class.

Reply:
Please check [42d08a7](https://github.com/nightwatchjs/nightwatch/pull/3566/commits/42d08a7462d27c609843117d87538ab2350bc231).

I couldn't think of any other good way of doing it, since I need all the functionality of `LocateElement.create()` method except just the last line where it converts `{using: 'id', value: 'org.wikipedia:id/search_container'}` to `{using: 'css selector', value: '*[id="org\\.wikipedia\\:id\\/search_container"]'}`.

Other way could be create a sub-class of `By` itself, but then we'd have to somehow pass the value of `isAppiumClient` into `By` class for that to work.

Reply:
Please check the implementation now. Also tested it by running `.waitForElementVisible` command on both Appium Client and normal desktop Chrome browser.